WcaOnRails/app/views/admin/post_results.html.erb

```diff
@@ -0,0 +1,129 @@
+<% provide(:title, "Post competition results & scrambles") %>
+<%= render layout: 'competitions/nav' do %>
+  <h1><%= yield(:title) %></h1>
+
+  <% if @existing_data.values.all?(&:zero?) %>
+    <%= alert :info do %>
+      This competition has no data yet. <%= link_to "Upload something", competition_admin_upload_results_edit_path(@competition.id) %> to get started.
+    <% end %>
+  <% else %>
+    <% if @existing_data[:inbox_result] > 0 || @existing_data[:inbox_person] > 0 %>
+      <%= alert :info do %>
+        <p>This competition is in the process of having data uploaded. If necessary, you can:</p>
+        <ul>
+          <% if @existing_data[:inbox_result] > 0 %>
+            <li>clear <%= link_to "InboxResults", competition_admin_delete_inbox_results_path(@competition.id) %></li>
```

Comment:
Would you mind adding a confirmation for links which lead to destructive actions please?
I've definitely misclicked a lot of time and been saved by such confirmation :p

As far as the implementation is concerned I believe it's a matter of simply adding `data: { confirm: "some string" }` to add in the `link_to` :)

Reply:
Makes sense, I've added some straight forward confirmation messages.